packages/functional-tests/tests/settings/misc.spec.ts

```diff
@@ -5,68 +5,62 @@
 import { test, expect } from '../../lib/fixtures/standard';
 
 test.describe('severity-1 #smoke', () => {
-  // https://testrail.stage.mozaws.net/index.php?/cases/view/1293407
-  test('removing secondary emails #1293407', async ({
+  test('removing secondary emails', async ({
     credentials,
     pages: { settings, secondaryEmail },
   }, { project }) => {
     test.slow(project.name !== 'local', 'email delivery can be slow');
+
     await settings.goto();
-    await settings.secondaryEmail.clickAdd();
+    await settings.secondaryEmail.addButton.click();
     const newEmail = credentials.email.replace(/(\w+)/, '$1_alt');
-    await secondaryEmail.addAndVerify(newEmail);
-    await settings.waitForAlertBar();
-    await settings.closeAlertBar();
-    await settings.secondaryEmail.clickDelete();
-    await settings.waitForAlertBar();
+    await secondaryEmail.addSecondaryEmail(newEmail);
+
+    await expect(settings.alertBar).toHaveText(/successfully added/);
+
+    await settings.alertBarDismissButton.click();
+    await settings.secondaryEmail.deleteButton.click();
+
     await expect(settings.alertBar).toHaveText(/successfully deleted/);
-    await settings.secondaryEmail.clickAdd();
-    await secondaryEmail.setEmail(newEmail);
+
+    await settings.secondaryEmail.addButton.click();
+    await secondaryEmail.emailTextbox.fill(newEmail);
     await secondaryEmail.submit();
+
     // skip verification
     await settings.goto();
-    expect(await settings.secondaryEmail.statusText()).toContain('UNCONFIRMED');
-    await settings.secondaryEmail.clickDelete();
-    await settings.waitForAlertBar();
+
+    await expect(settings.secondaryEmail.unverifiedText).toHaveText(
+      'unconfirmed'
+    );
+
+    await settings.secondaryEmail.deleteButton.click();
+
     await expect(settings.alertBar).toHaveText(/successfully deleted/);
   });
 
-  // https://testrail.stage.mozaws.net/index.php?/cases/view/1293504
-  test('settings help link #1293504', async ({ pages: { settings } }) => {
+  test('settings help link', async ({ pages: { settings } }) => {
     await settings.goto();
     const helpPage = await settings.clickHelp();
-    expect(helpPage.url()).toContain('https://support.mozilla.org');
-  });
 
-  // https://testrail.stage.mozaws.net/index.php?/cases/view/1293480
-  test('edit email communication preferences #1293480', async ({
```

Comment:
We should make sure we have coverage for this scenario, either via QA or automated testing. Filed https://mozilla-hub.atlassian.net/browse/FXA-9425